tests/unit/CMakeLists.txt

```diff
@@ -409,6 +409,13 @@ seastar_add_test (simple_stream
 seastar_add_app_test (smp
   SOURCES smp_test.cc)
 
+if(NOT Seastar_DPDK)
+  # disabled if built with DPDK, see seastar#2003
+  seastar_add_test (app-template
+    KIND BOOST
```

Comment:
It's better to do this in the test case itself, in case we add more tests to app-template_test.cc which can work with dpdk.

Reply:
Plus, less cmake = better

Reply:
@avikivity i explained the rationales in the commit message, why i cannot disable it in the test. could you please take a look? see https://github.com/scylladb/seastar/pull/2000/commits/b1ddb7ff94be7fb6ee282f14ff3f8375370e5e77

Reply:
i will be able to revert the cmake change once #2001 gets merged.

Reply:
this test now conflicts with the latest revision of #2001. i will rebase it once #2001 is merged.